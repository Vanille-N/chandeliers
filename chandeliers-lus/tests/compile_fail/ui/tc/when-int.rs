//! Wrong type for a clock.
chandeliers_lus::decl! {
    node bar(b : int; x : int when b) returns ();
    let tel;
}

fn main() {}
