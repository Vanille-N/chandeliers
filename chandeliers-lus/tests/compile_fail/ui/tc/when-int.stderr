error: Clock should be of type bool
 --> tests/compile_fail/ui/tc/when-int.rs:3:31
  |
3 |     node bar(b : int; x : int when b) returns ();
  |                               ^^^^^^
  |
note: The argument is found to be of type int
 --> tests/compile_fail/ui/tc/when-int.rs:3:18
  |
3 |     node bar(b : int; x : int when b) returns ();
  |                  ^^^
